tests/ui/same_item_push.rs

```diff
@@ -50,6 +50,16 @@ fn main() {
         //~^ ERROR: it looks like the same item is being pushed into this Vec
     }
 
+    #[clippy::msrv = "1.81"]
```

Comment:
Please make sure to also add the lint to the configuration:

https://github.com/rust-lang/rust-clippy/blob/25d319d1b2536e4c22b8013b95bd140a23ed78d7/clippy_config/src/conf.rs#L649-L651

Reply:
I always forget that users may configure the msrv in Clippy as well, thanks for the reminder!

Reply:
If you want a project, you can try to find a way to do this automatically. This has been something like a holly grail for some time. ^^